src/components/ui/AuthModal/SignUp/validationSchema.ts

```diff
@@ -0,0 +1,31 @@
+import { TFunction } from 'i18next';
+import * as Yup from 'yup';
+
+const getValidationSchema = (t: TFunction<'auth'>) =>
+    Yup.object({
+        fullName: Yup.string()
+            .trim()
+            .min(3, t('minNameMsg'))
+            .max(50, t('maxNameMsg'))
+            .required(t('nameRequired')),
+
+        email: Yup.string()
+            .trim()
+            .email(t('invalidEmailMsg'))
+            .required(t('emailRequired')),
```

Comment:
email format validation ?

Reply:
I have resolved it in the recent commit.